File: Cristian Wigand tarea repositorio/Ejercicio 1.cpp
#include<stdio.h>

class circulo{
	public:
		float radio;
		circulo(float radio1){
			radio=radio1;
		};
    	float calcularArea(){
	    	return 3.14*(radio*radio);
	    };
	    float calcularPerimetro(){
		    return 6.28*radio;
	    };
        float cambiarRadio(float radio2){
    	    return radio=radio2;
	    };
};

int main(){
	float n;
	printf("Ingresar radio del circulo: ");
	scanf("%f",&n);	
    circulo pelota(n);
    
    printf("El area del circulo es: %.2f \nEl perimetro del circulo es: %.2f",pelota.calcularArea(),pelota.calcularPerimetro());
    pelota.cambiarRadio(5.0);
    printf("\nEl nuevo valor del radio es: %.2f",pelota.radio);
    printf("\nEl area del circulo es: %.2f \nEl perimetro del circulo es:%.2f",pelota.calcularArea(),pelota.calcularPerimetro());
	
	return 0;
}

File: Cristian Wigand tarea repositorio/Ejercicio 1.py
#Ejercicio 1:
#Crea una clase Circulo.
#Define un atributo llamado radio.
#Define los siguientes metodos:

#Define un metodo constructor el cual inicialice el valor del atributo llamado radio.
#calcularArea() -> Este metodo retorna el area del circulo.
#calcularPerimetro() -> Este metodo retorna el perimetro del circulo.
#cambiarRadio() -> Este metodo permite cambiar el radio del circulo.
#Fuera de la clase, instancia un objeto de clase Circulo, asignandole un radio inicial.

#Muestra por pantalla el area de ese circulo.
#Muestra por pantalla el perimetro de ese circulo.

class Circulo:

    # Constructor
    def __init__(self, radio):
        # Atributos del objeto (tambien conocidos como atributos de instancia)
        self.radio = radio

    # Metodo
    def calcularArea(self):
        return 3.14*(self.radio**2)
    def calcularPerimetro(self):
        return 6.28*self.radio
    def cambiarRadio(self,radio):
        self.radio=radio

aux=float(input("Ingresar radio del circulo:"))
    
pelota = Circulo(aux)
print("El area del circulo es: ",pelota.calcularArea(),"\nEl perimetro del circulo es:",pelota.calcularPerimetro())
pelota.cambiarRadio(5)
print("El nuevo valor del radio es:",pelota.radio)
print("El area del circulo es: ",pelota.calcularArea(),"\nEl perimetro del circulo es:",pelota.calcularPerimetro())